Automattic/jetpack
# Cover block: paid-block-media-placeholder fix zero height

Fixes #22152 

#### Changes proposed in this Pull Request:

* Fixed the css to avoid overlapping

#### Jetpack product discussion NO



#### Does this pull request change what data or activity we track or use? NO



#### Testing instructions:





* checkout this branch and build project
* Create a new post on a free WP.com site
* Add Cover block
* Add any other block(s)
* See they're not hidding behind the Cover block placeholder as the paid-block-media-placeholder 

cc @ciampo @WunderBart
